### PR TITLE
Add onmerge action

### PR DESCRIPTION
**Pull request by akuhlman-cec**

### CODE REVIEW

File: script/binary [hunk 1]
```diff
@@ -17,7 +17,7 @@
 set -e
 
 
-GIT_COMMIT=$(git describe --tags --dirty)
+GIT_COMMIT=$(git rev-parse --short HEAD)
```
Comment (signed8bit) [inline script/binary:21]: Let's add a clarifying comment here on why we are changing what upstream had.

File: script/binary-controller [hunk 2]
```diff
@@ -42,7 +42,7 @@ else
 fi
 
 
-GIT_COMMIT=$(git describe --tags --dirty)
+GIT_COMMIT=$(git rev-parse --short HEAD)
```
Comment (signed8bit) [inline script/binary-controller:47]: Let's add a clarifying comment here on why we are changing what upstream had.

**Review by signed8bit — CHANGES_REQUESTED** [review]

File: script/binary-controller (hunk 2 above)
Comment (akuhlman-cec) [reply script/binary-controller:47]: removed change here since I'm actually no longer building this binary using this script

Comment (akuhlman-cec) [reply script/binary-controller:47]: actually I am, I put it back

**Pull request merged**